web-ui/src/main/resources/catalog/templates/admin/settings/system.html

```diff
@@ -53,7 +53,7 @@
                 translate}}</label>
               <!-- According to entry -->
               <div class="col-sm-8" data-ng-switch="s['@name']">
-
+								{{s['@name]'}}
```

Comment:
I can look at fixing this this morning

On Thu, Mar 12, 2015 at 7:49 AM, François Prunayre <notifications@github.com
> wrote:

> In web-ui/src/main/resources/catalog/templates/admin/settings/system.html
> <https://github.com/geonetwork/core-geonetwork/pull/854#discussion_r26281727>
> :
>
> > @@ -53,7 +53,7 @@
> >                  translate}}</label>
> >                <!-- According to entry -->
> >                <div class="col-sm-8" data-ng-switch="s['@name']">
> > -
> > +								{{s['@name]'}}
>
> It breaks the admin #856
> <https://github.com/geonetwork/core-geonetwork/issues/856>
> and if displayed we get the setting key on top of each field
> [image: image]
> <https://cloud.githubusercontent.com/assets/1701393/6613390/48311e00-c88c-11e4-8df2-4f866d2a42fe.png>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/geonetwork/core-geonetwork/pull/854/files#r26281727>.
>